open-api/rest-catalog-open-api.yaml

```diff
@@ -2747,6 +2747,54 @@ components:
         uuid:
           type: string
 
+    AzureCredentials:
```

Comment:
Is there no expiration for Azure credentials?

Reply:
there is, I didn't add it in the first iteration because I haven't double-checked the ADLS credentials